source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,189 @@
+3. For this tutorial, we will use some images of Frenck.
+   - Add the following lines into the `substitution` block.
+
+   {% raw %}
+
+     ```yaml
+      loading_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/loading_320_240.png
+      idle_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/idle_320_240.png
+      listening_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/listening_320_240.png
+      thinking_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/thinking_320_240.png
+      replying_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/replying_320_240.png
+      error_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/error_320_240.png
+    ```
+
+   {% endraw %}
+   
+4. Save the changes and select **Install**:
+   - Depending on your environment, the installation process can take a while.
+   - Expect this to take about 25 minutes. The code needs to be compiled before it can be installed.
+![ESP32-S3-BOX config file](/images/assist/esp32-s3-config-05.png)
+   
+5. Once the installation is complete, you can see the new image on the S3-BOX.
+   - Reboot the device.
+   - Now, speak a command to test the new setting. For example, *OK Nabu turn on the light*.
+
+## Using your own images
+
+Perform this task if you want to use your own images on the S32-S3-BOX.
+
+This task consist of 2 procedures:
+
+- [Preparing your own images](#to-prepare-your-own-images)
+- [Adding your images to the configuration](#to-add-your-images-to-the-configuration)
+  
+**Duration**: ca. 35 minutes, without counting the time to create the images.
+
+### About the images
+
+To make sure the images can be displayed properly on the S3-BOX screen, check out the following requirements.
+
+#### Illustrating 6 different states
+
+There are 6 different states to illustrate:
+
+- loading
+- idle
+- listening
+- thinking
+- replying
+- error
+
+#### Using light and dark image background
+
+To make it easier to distinguish the different states when you look at your screen, change the background depending on the state:
+
+- For loading and idle: use a dark background
+- For the listening, thinking, replying: use a light background
+- For error: As you like
+  
+You can also save the images with transparent background and then change the background in the configuration file.
+
+#### Image dimensions and file format
+
+- **Screen dimensions**: 320 x 240 pixels
```

Comment:
This is not a requirement. ESPHome compilation will automatically make it fit in this space. If it is not 4:3 ratio, it will show the background where there is no picture drawn.